<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDlg</class>
 <widget class="QDialog" name="MainDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Font Tool</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="fontList"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Style</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="styleSelection">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Char set</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="charsetSelection">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Base Size</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="spinBox">
           <property name="minimum">
            <number>6</number>
           </property>
           <property name="maximum">
            <number>200</number>
           </property>
           <property name="value">
            <number>128</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="selectCharsButton">
           <property name="text">
            <string>Select Characters</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushSave">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushClose">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushSave</sender>
   <signal>clicked()</signal>
   <receiver>MainDlg</receiver>
   <slot>onSaveFont()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushClose</sender>
   <signal>clicked()</signal>
   <receiver>MainDlg</receiver>
   <slot>onClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fontList</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>MainDlg</receiver>
   <slot>onFontSelected(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>styleSelection</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>MainDlg</receiver>
   <slot>onStyleSelected(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectCharsButton</sender>
   <signal>clicked()</signal>
   <receiver>MainDlg</receiver>
   <slot>onSelectCharacters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onFontSelected(QListWidgetItem*)</slot>
  <slot>onSaveFont()</slot>
  <slot>onClose()</slot>
  <slot>onStyleSelected(QString)</slot>
  <slot>onSelectCharacters()</slot>
 </slots>
</ui>
